commit 9cfd36156d04
Author: Francois Marier <francois@mozilla.com>
Date:   Fri Feb 2 13:30:28 2018 -0800

    Bug 1435098 - Gate flashinfobar list on the plugins.show_infobar. r=bytesized,gcp a=RyanVM
    
    The list of sites to suppress flash infobars on should not be
    downloaded from shavar unless the infobar feature is enabled.
    
    MozReview-Commit-ID: BjkS5vWiilg
    
    --HG--
    extra : source : 40f74605367ec4d620873fa8ff90c2dcc2a7ce31
---
 toolkit/components/url-classifier/SafeBrowsing.jsm | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git toolkit/components/url-classifier/SafeBrowsing.jsm toolkit/components/url-classifier/SafeBrowsing.jsm
index 64b8dd35de03..29d2a46fe72c 100644
--- toolkit/components/url-classifier/SafeBrowsing.jsm
+++ toolkit/components/url-classifier/SafeBrowsing.jsm
@@ -66,6 +66,7 @@ this.SafeBrowsing = {
     Services.prefs.addObserver("privacy.trackingprotection", this);
     Services.prefs.addObserver("urlclassifier", this);
     Services.prefs.addObserver("plugins.flashBlock.enabled", this);
+    Services.prefs.addObserver("plugins.show_infobar", this);
 
     this.readPrefs();
     this.addMozEntries();
@@ -151,7 +152,7 @@ this.SafeBrowsing = {
   blockedEnabled:       false,
   trackingAnnotations:  false,
   flashBlockEnabled:    false,
-  flashInfobarListEnabled: true,
+  flashInfobarListEnabled: false,
 
   phishingLists:                [],
   malwareLists:                 [],
@@ -231,6 +232,7 @@ this.SafeBrowsing = {
     this.blockedEnabled = Services.prefs.getBoolPref("browser.safebrowsing.blockedURIs.enabled");
     this.trackingAnnotations = Services.prefs.getBoolPref("privacy.trackingprotection.annotate_channels");
     this.flashBlockEnabled = Services.prefs.getBoolPref("plugins.flashBlock.enabled");
+    this.flashInfobarListEnabled = Services.prefs.getBoolPref("plugins.show_infobar", false);
 
     let flashAllowTable, flashAllowExceptTable, flashTable,
         flashExceptTable, flashSubDocTable,
